packages/react-sdk/src/css/preset-style.ts

```diff
@@ -0,0 +1,52 @@
+import { useMemo, useState } from "react";
+import { createValueContainer, useValue } from "react-nano-state";
+import type { PresetStyle } from "../db";
+import { getComponentMeta, getComponentNames } from "../components";
+import type { StyleProperty } from "@webstudio-is/css-data";
+
+const presetStyleContainer = createValueContainer<PresetStyle>([]);
+
+export const useSetPresetStyle = (presetStyle: PresetStyle) => {
+  useState(() => {
+    presetStyleContainer.value = presetStyle;
+  });
+};
+
+export const usePresetStyle = () => {
```

Comment:
```suggestion
export const usePresetStyles = () => {
```

Reply:
essentially everywhere where its not a style block like declaration for a single rule, please use `styles`